MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "jess@linux.com"
```

Comment:
but i contributed to docker before I worked here

is this somethinng we are getting into here, its totally seperate from this
pr, i will just revert that change

On Thu, May 21, 2015 at 1:00 PM, Brian Goff <notifications@github.com>
wrote:

> In MAINTAINERS
> <https://github.com/docker/docker/pull/13391#discussion_r30840911>:
>
> > @@ -591,7 +598,7 @@ made through a pull request.
> >
> >  	[people.jfrazelle]
> >  	Name = "Jessie Frazelle"
> > -	Email = "jess@docker.com"
> > +	Email = "jess@linux.com"
>
> but employment does not equate to maintainership... but those who work at
> Docker and those who no longer work at Docker.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13391/files#r30840911>.
>